src/app/component/Minimap/MiniMap.jsx

```diff
@@ -0,0 +1,80 @@
+/*** IMPORTS ***/
+// Module imports
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import GoogleMapReact from "google-map-react";
+import styled from 'styled-components';
+/*** [end of imports] ***/
+
+
+//size variations for maps
+const SmallMap = styled.div`
```

Comment:
style  should go into MiniMap.style.js

Reply:
Moved to new file called MiniMap.style.js